tests/testthat/test-vivax.R

```diff
@@ -20,7 +20,62 @@ test_that('Test difference between falciparum and vivax parameter lists', {
   vivax_parameters <- get_parameters(parasite = "vivax")
   
   expect_true(all(names(falciparum_parameters)[!names(falciparum_parameters) %in% names(vivax_parameters)] %in%
-               c("du","rvm","rva","rb","b0","b1","ib0","kb","theta0","theta1","kv","fv0","av","gammav","iv0","fd0","ad","gammad","d1","id0","kd","ub","uv","gamma1","pvm","init_ivm","init_ib","init_iva")))
+                    c("du","rvm","rva","rb","b0","b1","ib0","kb","theta0","theta1","kv","fv0","av","gammav","iv0","fd0","ad","gammad","d1","id0","kd","ub","uv","gamma1","pvm","init_ivm","init_ib","init_iva")))
   expect_true(all(names(vivax_parameters[!names(vivax_parameters) %in% names(falciparum_parameters)]) %in%
                     c("du_max","du_min","ku","au50","b","phi0lm","phi1lm","ic0lm","kclm","d0","ca","init_idm","f","gammal")))
 })
+
```

Comment:
In the existing tests there are mocked calls to schedule_infections in tests/testthat/test-infection-integration.R
I think adding to or copying the set up of some of those tests to check the vivax changes to that function would be helpful

Reply:
I gave adding a test like this a go - is this what you meant? I think it will make more sense to have this kind of test when subpatent infection is included.